.ci/pytorch/macos-test.sh

```diff
@@ -232,7 +232,7 @@ test_torchbench_smoketest() {
   local dtype=${dtypes[$1]}
   local models=(hf_T5 llama BERT_pytorch dcgan hf_GPT2 yolov3 resnet152 sam sam_fast pytorch_unet stable_diffusion_text_encoder speech_transformer Super_SloMo doctr_det_predictor doctr_reco_predictor timm_resnet timm_vovnet vgg16)
 
-  for backend in eager inductor; do
+  for backend in eager inductor export-aot-inductor; do
```

Comment:
Are you sure it wouldn't timeout? Jobs already take 2+ hours to finish, adding one more backend can push them over the limit